src/Components/Header/Header.css

```diff
@@ -1,13 +1,23 @@
+.logo {
+  display: flex;
+  align-items: center;
+  text-align: center;
+  justify-content: center;
+  border: 1px solid white;
+  font-size: 45px;
+  margin: 20px;
+  padding: 5px;
+  padding-top: 0;
+  height: 5vh;
+  width: 10vw;
+  border-radius: 65%;
+  font-family: 'Playfair Display SC', serif;
+  color: white;
+}
```

Comment:
thanks for working on the header, excited to see what it looks like

Reply:
+1 - excited to lock in styling today!